cognite/client/_api/functions.py

```diff
@@ -418,7 +446,7 @@ def retrieve_multiple(
         ids: Sequence[int] | None = None,
         external_ids: SequenceNotStr[str] | None = None,
         ignore_unknown_ids: bool = False,
-    ) -> FunctionList | Function | None:
```

Comment:
This is a bug, the _retrieve_multiple with list input cannot return `Function` or `None`